### DISCUSSION TITLE
Discretized mixture of logistics loss from PixelCNN++

### DISCUSSION THREAD

**Discussion by seunggs** [Lightning-AI/pytorch-lightning#6189] [code help: CV]
Does anyone understand why there seems to be 4 different parameters for discretized mixture of logistics loss implemented by OpenAI (line 46, https://github.com/openai/pixel-cnn/blob/master/pixel_cnn_pp/nn.py)? I thought it should be the mixture weights, mean, and scale of logistic distributions. Isn't the logit_probs the mixture weights? If so, what is coeffs?

I've been struggling for a few days now to understand this function so that I can implement it in PyTorch. Any help would be greatly appreciated!
